From 6259779e26c928d59b7aae66273da095fcfcb49b Mon Sep 17 00:00:00 2001
From: Hangbin Liu <liuhangbin@gmail.com>
Date: Tue, 6 Jul 2021 19:11:43 +0800
Subject: [PATCH] configure: update obsoleted m4 macros

Replace obsoleted AM_CONFIG_HEADER[1] with AC_CONFIG_HEADERS.
Also replace deprecated AM_PROG_LIBTOOL[2] with LT_INIT.

[1] https://www.gnu.org/software/automake/manual/html_node/Optional.html
[2] https://www.gnu.org/software/libtool/manual/html_node/LT_005fINIT.html

Signed-off-by: Hangbin Liu <liuhangbin@gmail.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2a840cc..3671f27 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,10 +1,10 @@
 AC_INIT(dropwatch,1.5.3)
 AC_PREREQ(2.12)dnl
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign] [subdir-objects])
-AM_PROG_LIBTOOL
+LT_INIT
 AC_SUBST(LIBTOOL_DEPS)
 
 AC_PROG_CC
-- 
2.31.1

